index.js

```diff
@@ -2,14 +2,14 @@
  * Objection related code
  */
 const Knex = require("knex");
-const knexConfig = require("./knexfile");
+const knexConfig = require("./knexfile.js");
```

Comment:
Why the suffix? _Just to know if there is something I need to keep in mind_

Reply:
Yeah, this is something I started doing but then dropped because is not a practice everyone is doing. 

The ESM spec for Node (link https://github.com/nodejs/node/blob/master/doc/api/esm.md), which is now available in Node 13 (link https://medium.com/the-node-js-collection/esm-in-node-has-been-unflagged-other-node-js-updates-of-this-week-46-2019-d805b5d4f93b) removes the ambiguity about importing a directory and a file, so you need to include now the file extension for this to work.

We can start doing it as a good practice and make it easier to migrate to ESM from CommonJS.